ethexe/processor/src/handling/events.rs

```diff
@@ -151,12 +151,24 @@ impl Processor {
 
     pub(crate) fn handle_wvara_event(
         &mut self,
-        _in_block_transitions: &mut InBlockTransitions,
+        in_block_transitions: &mut InBlockTransitions,
         event: WVaraEvent,
     ) -> Result<()> {
         match event {
-            WVaraEvent::Transfer { .. } => {
-                log::debug!("Handler not yet implemented: {event:?}");
+            WVaraEvent::Transfer { from, to, value } => {
+                if let Some(state_hash) = in_block_transitions.state_of(&to) {
+                    if in_block_transitions.state_of(&from).is_none() {
+                        let new_state_hash = self.db.mutate_state(state_hash, |_, state| {
+                            state.balance += value;
```

Comment:
it does to.balance += value, but what about from.balance -= value?

Reply:
As todo left, there needs additional testing and checking invariants on sendings from external user or local programs. Mostly because we need to also move its balance from router: coming up soon